### ISSUE TITLE
Simulator accessibility issues

### ISSUE THREAD

**Issue by microbit-robert** [microbit-foundation/python-editor-next#947]
Screen readers:

- The sound level thresholds are read as "clickable".
- It is not clear if the reader is reading messages sent from the micro:bit, or messages sent to the micro:bit by the user.
- The compass heading slider in the expanded compass module is missing a label. The reader reads "slider 0", rather than "compass heading slider 0".
- The reader reads the compass heading slider value in the collapsed compass module twice. E.g., "compass heading slider 00". Edit: This is because the tooltip is being read as well as the input value separately.

**Comment by microbit-robert** [comment]
Closed by #948.
